script/git-hooks/pre-commit

```diff
@@ -8,6 +8,11 @@
 # To enable, symlink this file to '.git/hooks/pre-commit' like so:
 #   ln -s ../../script/git-hooks/pre-commit .git/hooks/pre-commit
 
-files=$(git diff --name-only HEAD --diff-filter=d | grep '\.\(cpp\|h\)$')
-python script/validators/source_validator.py --files $files
-exit $?
+files=$(git diff --name-only HEAD --cached --diff-filter=d | grep '\.\(cpp\|h\)$')
+echo $files
```

Comment:
You can remove this `echo` since all scanned files are printed in the `source_validator.py` script.

Reply:
oops, that one is left from debugging